.github/workflows/ci.yml

```
@@ -0,0 +1,84 @@
+name: Tests and PyPI publishing
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Run tests Python ${{ matrix.python }}, Django ${{ matrix.django }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.8', '3.9', '3.10', '3.11']
+        django: ['3.2', '4.0', '4.1', '4.2']
```

```suggestion
      matrix:
        python: ['3.8', '3.9', '3.10', '3.11']
        django: ['3.2', '4.1', '4.2']
        exclude:
          - python: '3.11'
            django: '3.2'
```
* Django 3.2 does not support Python 3.11
* Django 4.0 is end of life and no longer maintained -> we don't support it either